server/src/main/java/io/druid/indexing/overlord/autoscaling/ResourceManagementSchedulerConfig.java

```diff
@@ -28,6 +28,9 @@
 public class ResourceManagementSchedulerConfig
 {
   @JsonProperty
+  private String region = "us-east-1";
```

Comment:
I believe region should be configured as part of `EC2EnvironmentConfig`. The `ResourceManagementScheduler` is agnostic of the underlying autoscaler and should not depend anything AWS specific.